tests/test_schema.py

```diff
@@ -624,9 +1063,45 @@ def test_filter_non_existing_registry(self):
                                                      self.timestamp.isoformat())
         executed = client.execute(test_query)
 
-        transactions = executed['data']['transactions']
+        transactions = executed['data']['transactions']['entities']
         self.assertListEqual(transactions, [])
 
+    def test_pagination(self):
+        """Check whether it returns the transactions searched when using pagination"""
+        timestamp = datetime_utcnow()
```

Comment:
Leave an empty line in between this two.

Reply:
Done!